DESIGN.md

```diff
@@ -0,0 +1,34 @@
+# Design Doc 
+
+link: [DESIGN.md](DESIGN.MD) 
+
+## Objective
+
+To Create a new Notification system in TypeScript to provide users with better feeds and more features. 
+
+## Goals
+
+Our current notification feed has many limitations. It is specific to a selected archive and doesn’t include all the notifications generated in the system.
+Also our current notifications don’t have actionable buttons which can help users navigate easily in the app and save a lot of time.
+The proposed notification system will be able to eliminate all the limitations of the current notification system and also be able to provide users with better features like 
+actionable buttons. 
+
+## Overview
+
+Previously we have been displaying notifications for each archive and hence the notifications feeds were also configured to fetch data based on the archive. Our design has never 
+considered providing users with an ability to take action right from the notification feed. In recent years, new features haven't been rolled out with accompanying notifications, 
+which has left the notifications feed limited in the types of notices it displays to users. 
+@Bryson new design has combined all feeds into one account notification feed, and also has actionable buttons corresponding to each feed. The limitations of the current 
+notification system calls for a redesign and re-architecture. 
+
+## Detailed Design
+
+We will create a new microsystem, preferably in TypeScript and a new database , the chosen one is PostgreSQL.  Microsystem will make it  easier to open source the new service in 
```

Comment:
The term of art is [microservice](https://en.wikipedia.org/wiki/Microservices); let's search-and-replace all the occurrences of "microsystem". :slightly_smiling_face:

Reply:
> The term of art is [microservice](https://en.wikipedia.org/wiki/Microservices); let's search-and-replace all the occurrences of "microsystem". 🙂

Ah thank you.. My bad